<launch>

	<node pkg="ublox" type="ublox_node" name="uBlox_GPS" output="screen" respawn="true"/>

</launch>
